docs/research/wallet-comparison.md

```diff
@@ -0,0 +1,171 @@
+## Existing Wallets
+
+List of Lightning wallet to be analysed:
+
+- Wallets to connect to your existing node
+  - [BlueWallet](https://bluewallet.io/)
+    - Needs LNDHub
+  - [Zeus](https://zeusln.app/)
+    - Can connect to LND or C-Lightning
+  - [FullyNoded](https://fullynoded.app/)
+    - Can connect to LND or C-Lightning
+- Full nodes
+  - [Breez](https://breez.technology/)
+    - Uses a forked LND version
+  - [Phoenix](https://github.com/ACINQ/phoenix)
+    - Uses a custom lighting implementation
+
+Below we have a look at these wallets and their feature set.
+If a wallet is not mentioned below it has not been analysed (yet).
+
+### Phoenix Wallet
+
+In a nutshell: Non-custodial setup and a great UX that comes with some (minimal) trust for certain features.
+
+Setup:
+
+- Non-custodial setup
+  - Run own LND node on Phone ([Eclair](https://github.com/ACINQ/eclair), ACINQ in-house lightning implementation)
+  - Channel management through ACINQ nodes
+  - Can configure own Electrum Server
+  - Cannot configure own Lightning node (outside the phone setup)
+- Where is trust involved?
+  - Initial channel opening
+  - Onboarding with BTC
+
+Pro:
+
+- Good looking and efficient UI and UX
+- One of the best onboarding UX out there
+  - One-click wallet setup
+  - Onboarding using BTC
+- BIP39 (+BIP84) 12-word wallet backup
+
+Pro/Con:
+
+- Channels management by ACINQ
+  - Slightly higher costs for more convenience
+- Channel backup managed by ACINQ
+  - "Minimal" trust for more convenience
+
+Con:
+
+- Hard to understand the underlying constraints for the Lightning setup
+- Costs for transactions are not transparent (retrospectively)
+  - Example
+    - Initially funded the wallet via BTC, had BTC worth `50.3` USD incoming.
+    - Once they were actually confirmed it changed to `49.95` USD because the fees were deducted.
+    - These fees are never visible again, when going to the details I cannot see how much fee was actually deducted and what for.
+    - A better cost breakdown would be nice to have.
+
+For details check these links:
+
+- https://medium.com/@ACINQ/introducing-phoenix-5c5cc76c7f9e
+- https://phoenix.acinq.co/faq
+
+### Blue Wallet
+
+In a nutshell: Semi- or Non-custodial (when using own LND) setup and a good UX that includes multiple wallet, but overall not as good as Phoenix.
+
+Setup:
+
+- Plug in own LND node, or use [LndHub](https://github.com/BlueWallet/LndHub)
+  - LndHub is a semi-custodial setup ("minimum trust for endusers")
+- Where is trust involved?
+  - LndHub
+
+Pro:
+
```

Comment:
IMHO a big pro: LApps marketplace. many, many features enabled thanks to that
```suggestion
- LApps marketplace
```

Reply:
OMG that feature is well hidden (It's a button `Marketplace` in the wallet view) 😅

I'll add that, just did not notice it before!

Reply:
well, it's inside a wallet, because you use that wallet funds with the marketplace. FWIW, in my case the button being right there in the middle of the screen was pretty easy to find ;D